include/wlr/backend.h

```diff
@@ -57,5 +57,10 @@ struct wlr_session *wlr_backend_get_session(struct wlr_backend *backend);
  * Returns the clock used by the backend for presentation feedback.
  */
 clockid_t wlr_backend_get_presentation_clock(struct wlr_backend *backend);
+/**
+ * Returns the file descriptor that is internally opened for rendering.
```

Comment:
I think it would be good to explicitly spell out that this is a DRM node file descriptor. And also that the FD is owned by the backend, the caller must not close it.

Reply:
What do you think of the following text:

"Returns the DRM node file descriptor used by the backend's underlying platform. Can be used by consumers for additional rendering operations. The consumer must not close the file descriptor since the backend continues to have ownership of it."

Reply:
LGTM!